comments.html

```diff
@@ -0,0 +1,274 @@
+                  </div>
+                  <div class="content">
+                    <img src="" alt="" />
+                    <div class="text">
+                      <h5 class="mb-o">Zoe Bowman</h5>
+                      <div class="rating-wrap">
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="mt-3">
+                    <p>
+                      Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+                      Qui expedita ullam facilis eaque vitae consectetur
+                      excepturi fugiat modi voluptate sit?
+                    </p>
+                  </div>
+                </blockquote>
+                <blockquote class="border-bottom">
+                  <div class="float-end">
+                    <div class="btn-group">
+                      <button class="btn btn-light btn-sm" data-bs-toggle="tooltip" data-bs-title="Gefällt mir">
+                        <i class="fa fa-thumbs-up"></i>
+                      </button>
+                      <button class="btn btn-light btn-sm" data-bs-toggle="tooltip" data-bs-title="Gefällt mir nicht">
+                        <i class="fa fa-thumbs-down"></i>
+                      </button>
+                    </div>
+                    <button class="btn btn-light btn-sm">
+                      <i class="fa fa-ellipsis-v"></i>
+                    </button>
+                  </div>
+                  <div class="content">
+                    <img src="" alt="">
+                    <div class="text">
+                      <h5 class="mb-0">Bryan Hernandez</h5>
+                      <div class="rating-wrap">
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                        <span class="fa fa-star checked"></span>
+                      </div>
+                    </div>
+                  </div>
+                  <div class="mt-3">
+                    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Molestias, nemo quisquam? Illo molestiae iusto facilis.</p>
+                  </div>
+
+                </blockquote>
+              </div>
+            </div>
+            <div class="card">
+              <div class="card-body">
+                <h4 class="mb-3">Yorum Yap</h4>
+                <form>
+                  <div class="mb-3">
+                    <input type="text" class="form-kontrol" placeholder="Name">
```

Comment:
Ein kleiner Tippfehler macht dir hier etwas kaputt:

```suggestion
                    <input type="text" class="form-control" placeholder="Name">
```